backends/platform/sdl/macosx/macosx.cpp

```diff
@@ -261,6 +261,21 @@ Common::String OSystem_MacOSX::getDefaultLogFileName() {
 	return Common::String(prefix) + "/Library/Logs/scummvm.log";
 }
 
+Common::String OSystem_MacOSX::getDefaultIconPath() {
```

Comment:
It is called everywhere icon***S***path. So, perhaps rename it to `getDefaultIconsPath()` for consistency

Reply:
Renamed.